tests/Subscriber/AutoUpdateTitleWithLabelSubscriberTest.php

```diff
@@ -59,7 +59,7 @@ public function testOnPullRequestLabeled()
 
         $this->assertCount(2, $responseData);
         $this->assertSame(1234, $responseData['pull_request']);
-        $this->assertSame('[Console][FrameworkBundle] [bar] Foo', $responseData['new_title']);
+        $this->assertSame('[Console][FrameworkBundle][bar] Foo', $responseData['new_title']);
```

Comment:
Or is this really expected? I mean I often use `[Mailer][TheNameOfTheMailerBridgeHere] Add feature`, so IMHO it belongs to the "labels"

Reply:
So you agree here @xabbuh ?

Reply:
No, I agree with you on this.